empire.go

```diff
@@ -425,6 +425,9 @@ type DeploymentsCreateOpts struct {
 	// Image is the image that's being deployed.
 	Image image.Image
 
+	// Environment is the environment where the image is being deployed
+	Environment string
```

Comment:
This is probably ok for now, but it'd probably be better in the long run if we had this on the Empire struct itself, so it works for non-github deployments. If I `emp deploy <image>`, I'd still expect to see the environment in the event.

That means we'd need an `--environment` flag.